components/ResourceCounts.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from "react";
+import { Badge, Button } from "@mantine/core";
+import { cleanNotifications, showNotification } from "@mantine/notifications";
+
+export interface ResourceCountResponse {
+  [x: string]: number;
```

Comment:
```suggestion
  [resourceType: string]: number;
```

Naming the key `resourceType` could help to make this easier to understand what it is representing.